anchore-ci/utils

```diff
@@ -1,18 +1,24 @@
 #!/usr/bin/env bash
 
-#########################################################
-###   COMMON UTILITY FUNCTIONS - ALPHABETICAL ORDER   ###
-#########################################################
+#############################################################
+### Common utility functions
+#############################################################
+
+export BASH_DEBUG=""
+
+bash_debug() {
```

Comment:
What's the purpose of this rather then using the print_colorized function?

Reply:
Oh I see the point, it's optional. Maybe that conditional should check for the VERBOSE variable? And maybe this could just be wrapped into print_colorized with a DEBUG color_code?

Reply:
I'm not opposed to that, but I'm not crazy about it either.

Reply:
How about keeping it a separate function, but making it conditional on VERBOSE?

Reply:
I feel that they're for different things... like, I'd want to use a conditional print/debug statement without also having to see every command printed.  I'd also be fine removing it from the code and just using echo/printf when I need to debug locally.

Reply:
Fair enough. The bash_debug makes sense in that respect. It can stay how it is!